_pages/en_US/3dsropxploitinjector-(kartminer7).txt

```diff
@@ -0,0 +1,72 @@
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+### What You Need
+
+* A physical or digital copy of Mario Kart 7 updated to the latest version (v1.2)
+  * You can update Mario Kart 7 from eShop or by following the update prompt when you launch the game
+* A Windows computer
+* Your `movable.sed` file from completing Seedminer
+* The latest release of [Kartminer7](https://github.com/zoogie/Kartminer7/releases/latest)
+
+#### Section I - Preparing Mario Kart 7 data
+
+In this section, you will have the 3DS generate some data in Mario Kart 7 that will be used to install the exploit in a later section.
+
+1. Insert your SD card into your console
+1. Open Mario Kart 7
+1. Tap the "Mario Kart Channel" button on the bottom
+1. If you are prompted to, hit "Next" and "OK" to all of the prompts that you see
+1. Tap the "StreetPass List" button on the bottom of the screen, in the middle
+1. Press the (Home) button to suspend Mario Kart 7
+1. Disable wireless connections by flicking the wireless switch on the side of the console or by navigating to HOME Menu Settings (top left) -> Wireless Communication -> OFF
+1. Power off your console
+
+#### Section II - Prep Work
+
+1. Insert your SD card into your computer
+1. Extract the Release_BETA_mk7 `.zip` to somewhere on your computer (i.e. your desktop)
+1. Copy your `movable.sed` file to the `resources` folder of the Release_BETA_mk7 folder
+  + Ensure that the file is named exactly `movable.sed`
+1. Open the `backup` folder inside of the Release_BETA_mk7 folder
+1. Double-click on the BACKUP `.bat` file corresponding to your console's region
+  + If you get a message from Windows Defender, click "More info", "Run anyway", then press any key to continue
+  + If the backup was successful, you should see a new folder appear
+  + If you wish to restore this backup at a later date (after finishing this guide), insert your SD card and double-click on the RESTORE `.bat` file corresponding to your console's region
+1. Go back to the Release_BETA_mk7 folder, then double-click on the `.bat` file corresponding to your console's region AND model
+1. Reinsert your SD card into your console
+
+#### Section III - Kartminer7
+
+1. Power on your console
+1. Open Mario Kart 7
+1. Tap the "Mario Kart Channel" button on the bottom
+1. Tap the "StreetPass List" button on the bottom of the screen, in the middle
+1. Press (A) over and over again
+1. If the exploit was successful, the **target 3DS** will have booted into the 3DS ROP xPloit Injector
+
+### Next steps
+
+If your shoulder buttons (L and R) are working, continue to [Installing boot9strap (3DS ROP xPloit Injector-USM)]
+{: .notice--primary}
+
+If your shoulder buttons (L and R) are NOT working, continue to [menuhax67 + nimhax (3DS ROP xPloit Injector)](menuhax67-nimhax-(3dsropxploitinjector))
```

Comment:
Is there any reason you would need _nimhax_ after kartminer7 - obviously you'd need menuhax67 (for a HBL entrypoint) but we already end up with a movable.sed as a result of needing seedminer to do kartminer7 so this just seems redundant.

(yes I'm aware this page is also linked for kartdlphax, and yes, kartdlphax would need nimhax in this case scenario)

Reply:
You're right, this probably is redundant.

Reply:
Among Us is a 2018 [online multiplayer](https://en.wikipedia.org/wiki/Online_multiplayer) [social deduction game](https://en.wikipedia.org/wiki/Social_deduction_game) developed and published by American game studio Innersloth. The game was inspired by the party game [Mafia](https://en.wikipedia.org/wiki/Mafia_(party_game)) and the [science fiction](https://en.wikipedia.org/wiki/Science_fiction_film) [horror](https://en.wikipedia.org/wiki/Horror_film) film [The Thing](https://en.wikipedia.org/wiki/The_Thing_(1982_film)). The game allows for [cross-platform play](https://en.wikipedia.org/wiki/Cross-platform_play), released on [iOS](https://en.wikipedia.org/wiki/IOS) and [Android](https://en.wikipedia.org/wiki/Android_(operating_system)) devices in June 2018 and on [Windows](https://en.wikipedia.org/wiki/Windows) later that year in November. The game was ported to the [Nintendo Switch](https://en.wikipedia.org/wiki/Nintendo_Switch) in December 2020 and on the [PlayStation 4](https://en.wikipedia.org/wiki/PlayStation_4), [PlayStation 5](https://en.wikipedia.org/wiki/PlayStation_5), [Xbox One](https://en.wikipedia.org/wiki/Xbox_One) and [Xbox Series X/S](https://en.wikipedia.org/wiki/Xbox_Series_X/S) in December 2021. A virtual reality adaptation, [Among Us VR](https://en.wikipedia.org/wiki/Among_Us_VR), was released on November 10, 2022.